File: 2022/OpenCV4 tutorial/1 Test OpenCV/just_test.py
import cv2 as cv
import numpy as np


def just_test():
    # 显示一张100x100全黑的图，窗口标题为"the title"
    cv.imshow("the title", np.zeros((100, 100, 3), dtype=np.uint8))
    # 按任意键结束显示
    cv.waitKey()
    # 关闭窗口
    cv.destroyAllWindows()

if __name__ == '__main__':
    just_test()

File: 2022/OpenCV4 tutorial/1 Test OpenCV/just_test.cs
using System;
using OpenCvSharp; //导入OpenCV4
namespace CVSharpApp1
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("This is a test for OpenCVSharp");
            // 显示一张100x100全黑的图，窗口标题为"the title"
            Cv2.ImShow("the title", Mat.Zeros(100, 100, MatType.CV_8UC3));
            // 按任意键结束显示
            Cv2.WaitKey();
        }/* end of static void Main */
    }
}
